### PR TITLE
[IZPACK-1698] Internationalization of CustomPanel is not available in SummaryPanel

### PR DESCRIPTION
**Pull request by hiteshbosamiya**

### CODE REVIEW

**Comment by reinhapa** [comment]
@hiteshbosamiya can you also provide a JUnit test for the enhanced behavior.

**Comment by hiteshbosamiya** [comment]
Added JUnit test for enhanced behavior.

File: izpack-compiler/src/main/java/com/izforge/izpack/compiler/CompilerConfig.java [hunk 1]
```diff
@@ -2194,7 +2194,7 @@ private void addResources(IXMLElement data, File baseDir) throws CompilerExcepti
 
             // Just validate to avoid XML parser errors during installation later
             if (id.startsWith(Resources.CUSTOM_TRANSLATIONS_RESOURCE_NAME)
-                    || id.startsWith(UserInputPanelSpec.LANG_FILE_NAME)
+                    || id.startsWith(Resources.USER_INPUT_TRANSLATIONS_RESOURCE_NAME)
```
Comment (akuhtz) [inline izpack-compiler/src/main/java/com/izforge/izpack/compiler/CompilerConfig.java:2197]: Can ```LANG_FILE_NAME``` then be removed from ```UserInputPanelSpec``` ?

**Review by akuhtz — CHANGES_REQUESTED** [review]

**Comment by hiteshbosamiya** [comment]
Removed `LANG_FILE_NAME` from `UserInputPanelSpec`.

File: izpack-compiler/src/main/java/com/izforge/izpack/compiler/CompilerConfig.java (hunk 1 above)
Comment (hiteshbosamiya) [reply izpack-compiler/src/main/java/com/izforge/izpack/compiler/CompilerConfig.java:2197]: Done

**Review by akuhtz — APPROVED** [review]

**Comment by hiteshbosamiya** [comment]
@reinhapa, can you please look into this PR now as it has been approved by @akuhtz? Thanks.

**Comment by reinhapa** [comment]
Closes #741

**Pull request merged**